package.json

```diff
@@ -173,7 +173,7 @@
     "cozy-harvest-lib": "6.10.0",
     "cozy-interapp": "0.6.2",
     "cozy-keys-lib": "3.8.0",
-    "cozy-konnector-libs": "4.30.3-beta.1",
```

Comment:
I don't like that. Are we sur that what we had in the beta was merged?

Reply:
This was introduced by https://github.com/cozy/cozy-banks/pull/1818 , which is two-years old, and already related to the categorization. So I assumed we should be good?